markets/spot-market/contracts/modules/AsyncOrderModule.sol

```diff
@@ -106,15 +106,6 @@ contract AsyncOrderModule is IAsyncOrderModule {
             SpotMarketFactory.TransactionType.ASYNC_SELL
         );
 
-        // Calculate fees
-        // TODO: use `amountUsable` instead of `synthAmountEscrowedUsd`?
-        (, int256 feesQuoted) = Fee.calculateFees(
-            marketId,
-            msg.sender,
-            synthAmountEscrowedUsd,
-            SpotMarketFactory.TransactionType.ASYNC_SELL
-        );
```

Comment:
the `feesQuoted` wasn't really being used here.  but lets revisit async orders in general later.  its a bit unclear what we should escrow and which price to honor (commitment vs settlement)

Reply:
In a sell commitment, I think we burn all of the escrowed synths, withdraw their value in usd, and run it through the fee collector to potentially deposit it all back.

We could probably add a conditional here to avoid the gas from the round trip as an optimization.

Reply:
let's chat tomorrow about this.  

as a reminder for tomorrow:  the usd amount that is escrowed could be more or less than the amount we'd need based on price during settlement.  lets go through this together; might be missing something.